ConsumePlugin/ConsumePlugin.fsproj

```diff
@@ -6,88 +6,80 @@
+      <WhippetParamPrivateTypeNoAttr>GenerateMock</WhippetParamPrivateTypeNoAttr>
+      <WhippetParamPrivateTypeInternalFalseNoAttr>GenerateMock(false)</WhippetParamPrivateTypeInternalFalseNoAttr>
+      <WhippetParamVeryPublicTypeNoAttr>GenerateMock</WhippetParamVeryPublicTypeNoAttr>
+      <WhippetParamCurriedNoAttr>GenerateMock</WhippetParamCurriedNoAttr>
+      <WhippetParamTypeWithInterfaceNoAttr>GenerateMock</WhippetParamTypeWithInterfaceNoAttr>
     </Compile>
     <Compile Include="Vault.fs" />
     <Compile Include="GeneratedVault.fs">
-      <MyriadFile>Vault.fs</MyriadFile>
+      <WhippetFile>Vault.fs</WhippetFile>
     </Compile>
     <Compile Include="SerializationAndDeserialization.fs" />
     <Compile Include="GeneratedSerde.fs">
-      <MyriadFile>SerializationAndDeserialization.fs</MyriadFile>
+      <WhippetFile>SerializationAndDeserialization.fs</WhippetFile>
     </Compile>
     <Compile Include="Catamorphism.fs" />
     <Compile Include="GeneratedCatamorphism.fs">
-      <MyriadFile>Catamorphism.fs</MyriadFile>
+      <WhippetFile>Catamorphism.fs</WhippetFile>
     </Compile>
     <Compile Include="FSharpForFunAndProfitCata.fs" />
     <Compile Include="GeneratedFileSystem.fs">
-      <MyriadFile>FSharpForFunAndProfitCata.fs</MyriadFile>
+      <WhippetFile>FSharpForFunAndProfitCata.fs</WhippetFile>
     </Compile>
     <Compile Include="List.fs" />
     <Compile Include="ListCata.fs">
-      <MyriadFile>List.fs</MyriadFile>
+      <WhippetFile>List.fs</WhippetFile>
     </Compile>
     <Compile Include="Args.fs" />
     <Compile Include="GeneratedArgs.fs">
-      <MyriadFile>Args.fs</MyriadFile>
+      <WhippetFile>Args.fs</WhippetFile>
     </Compile>
     <None Include="swagger-gitea.json" />
     <Compile Include="GeneratedSwaggerGitea.fs">
-      <MyriadFile>swagger-gitea.json</MyriadFile>
-      <MyriadParams>
-        <GenerateMockInternal>true</GenerateMockInternal>
-        <ClassName>Gitea</ClassName>
-      </MyriadParams>
+      <WhippetFile>swagger-gitea.json</WhippetFile>
+      <WhippetParamGenerateMockInternal>true</WhippetParamGenerateMockInternal>
+      <WhippetParamClassName>Gitea</WhippetParamClassName>
     </Compile>
     <Compile Include="Generated2SwaggerGitea.fs">
-      <MyriadFile>GeneratedSwaggerGitea.fs</MyriadFile>
+      <WhippetFile>GeneratedSwaggerGitea.fs</WhippetFile>
     </Compile>
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="RestEase" Version="1.6.4"/>
     <ProjectReference Include="..\WoofWare.Myriad.Plugins.Attributes\WoofWare.Myriad.Plugins.Attributes.fsproj" />
-    <ProjectReference Include="..\WoofWare.Myriad.Plugins\WoofWare.Myriad.Plugins.fsproj" PrivateAssets="all" />
-    <PackageReference Include="Myriad.Sdk" Version="0.8.3" PrivateAssets="all" />
+    <ProjectReference Include="..\WoofWare.Myriad.Plugins\WoofWare.Myriad.Plugins.fsproj" PrivateAssets="all" WhippetPlugin="true" />
```

Comment:
Now all you need to do is add `WhippetPlugin="true"` and that's all the plugin registration done.